.yarnrc.yml

```diff
@@ -33,3 +33,6 @@ packageExtensions:
     peerDependenciesMeta:
       "algoliasearch":
         optional: true
+  "json-ref-lite@*":
+    dependencies:
+      "sync-request": "*"
```

Comment:
Do we need it? From a cursory glance in `json-ref-lite`, it's only used in a particular case:

![image](https://user-images.githubusercontent.com/1037931/75783379-c44f5b00-5d60-11ea-9e1d-5abed20c7c72.png)

Reply:
~~You're right, it's not actually needed. I added it a few days ago to get rid of a weird error message that happened only in specific cases that I don't remember anymore and that I can't reproduce anymore.😅~~
As I was writing this, Netlify just made me remember.😄
```
Module not found: Error: A package is trying to access another package without   the second one being listed as a dependency of the first one
11:07:16 PM:   Required package: sync-request (via "sync-request")
11:07:16 PM:   Required by: json-ref-lite@npm:1.1.0
```

Reply:
If instead I do
```yaml
"json-ref-lite@*":
  peerDependenciesMeta:
    "sync-request":
      optional: true
```
I get:
```
Module not found: Error: A package is trying to access a peer dependency that should be provi
  ded by its direct ancestor but isn't
  Required package: sync-request (via "sync-request")
  Required by: json-ref-lite@virtual
```

Reply:
Indeed 🙁 This is because Webpack resolves the `require` call even if it's a dead call... annoying, but I guess that's the way to go.